resources/provider/migrations/timescale/22_v0.10.14.up.sql

```diff
@@ -1,42 +1,48 @@
 -- Resource: ec2.instance_statuses
-CREATE TABLE IF NOT EXISTS "aws_ec2_instance_statuses" (
```

Comment:
Should not edit migration files from previously tagged releases, this causes confusion down the line. Please revert this change.

Reply:
only formatting has changed

Reply:
I know but it still causes confusion when we're inspecting releases and fixing migrations.

Reply:
would remove changes for this file